code/controllers/subsystems/economy.dm

```diff
@@ -77,7 +76,7 @@ var/datum/controller/subsystem/economy/SSeconomy
 	department_account.account_number = next_account_number
 	next_account_number += rand(1,500)
 	department_account.remote_access_pin = rand(1111, 111111)
-	department_account.money = 5000
+	department_account.money = station_accounts[department]
```

Comment:
This variable name is somewhat ambiguous considering that we have a `department_accounts` list that holds the actual accounts.
It would imho be better to rename it to `department_funds`